x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_host_count.ts

```diff
@@ -32,6 +32,24 @@ export const useHostCount = () => {
       const filters: QueryDslQueryContainer = {
         bool: {
           ...query.bool,
+          must: [
+            {
+              bool: {
+                should: [
+                  {
+                    term: {
+                      'event.module': 'system',
+                    },
+                  },
+                  {
+                    term: {
+                      'metricset.module': 'system',
```

Comment:
This is added because of the older metricbeat versions (I checked the test data and added it after, details [here](https://github.com/elastic/kibana/issues/176403#issuecomment-1954232722))

Reply:
Can you add a comment to the code, please?

Reply:
Do we want to keep backward compatibility with beats < 8.0? Would users use beats < 8.0 with Kibana/ES > 8.0? 🤔

Reply:
Based on the [support matrix](https://www.elastic.co/support/matrix) we should (7.17 - 8.12), I think that's why we still keep testing also with 7.x data. Once we drop the support we can remove this line (I will add a comment as suggested)